data/Identifying_production_data_types.md

```diff
@@ -1,3 +1,216 @@
 # Data types found in production service repos
 
+---
 
+#Expression
+##Description this stores expression data of a variety of different types (microarray, RNA-Seq, Proteomics, q-PCR)
+
+##Methods
+These will need to be changed and rewritten.  Currently it is all written in Perl and acesses the data via direct SQL queries against the Central Store.Also many of the methods were geared for searching before search existed.  The search related methods will not be carried over and should be abandoned.  The currented outdated methods exist here: https://github.com/kbase/expression/blob/master/lib/Bio/KBase/KBaseExpression/KBaseExpressionImpl.pm
+I envision the following new methods being done and operating on the workspace objects:
+* Visualize Sample
+* Visualize Series
+* Add annotation
+* Get Data By SampleIDs and FeatureIDs
+* Visualize ReplicateGroup?
+* Not sure what the RNA-Seq people will need.  Currently there are no methods for them in this spec.
+* Visualize the expression data for a genome (by samples)
+* Visulaize the expression data for a list of features (by samples)
+* Compare Samples (define denominator as specific sample or average of a group of samples)
+* Average Samples together to make a representative Sample
+* Compare Samples vs default controls?
+* Determine ON and OFF calls
+* Get Top Changers
+* Make new sample with original values
+
+##Data
+The data is currently being stored in both the Central Store and the Workspace.  They are not dependent on one another.
```

Comment:
Can you call these out very explicitly?  What exactly is being accessed from the central store vs. the workspace?